.CI/create_feedstocks.py

```diff
@@ -141,8 +141,10 @@ def write_token(name, token):
                '[ci skip]'.format(', '.join(removed_recipes),
                          s=('s' if len(removed_recipes) > 1 else '')))
         if is_merged_pr:
-            subprocess.check_call(['git', 'remote', 'add', 'upstream_with_token',
-                                   'https://conda-forge-admin:{}@github.com/conda-forge/staged-recipes'.format(os.environ['GH_TOKEN'])])
+            # Capture the output, as it may contain the GH_TOKEN.
+            out = subprocess.check_output(['git', 'remote', 'add', 'upstream_with_token',
```

Comment:
This is to stop the GH_TOKEN leak as it did. No real harm done, but there was a short period of time where anybody had access to the conda-forge organisation. :scream: 

Token revoked, and we should be all tickety-boo.